src/craft/app/services/SLPluginsService.php

```diff
@@ -0,0 +1,8 @@
+<?php
+namespace Craft;
+
+class SLPluginsService extends PluginsService {
```

Comment:
Inheritance is different from monkey patching.

https://github.com/craftcms/cms/blob/2.6.2984/src/services/PluginsService.php
https://github.com/craftcms/cms/blob/2.6.2984/src/services/UpdatesService.php